subsys/can/Kconfig

```diff
@@ -0,0 +1,23 @@
+# Kconfig - CAN configuration options
+#
+# Copyright (c) 2019 Erwin Rol <erwin@erwinrol.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CAN_STACKS
+	bool "CAN support"
+	help
+	  This option enables CAN support.
+
+if CAN_STACKS
+
+module = CAN_STACKS
+module-str = can_stacks
+source "subsys/logging/Kconfig.template.log_config"
```

Comment:
Why do you need logging for CAN stacks?

Reply:
General CAN logging? I am not really familiar with the logging framework, not idea how fine that should be split up, maybe just a "CAN" logging is fine for all Canopen, devicnet, j1939, etc.